cmd/snap/cmd_connections.go

```diff
@@ -114,6 +115,22 @@ func (b byConnectionData) Less(i, j int) bool {
 	return iCon.slot < jCon.slot
 }
 
+func definingAttributeValue(conn *client.Connection) string {
+	var value string
+
+	switch conn.Interface {
+	case "content":
+		value, _ = conn.PlugAttrs["content"].(string)
+		if value == "" {
```

Comment:
Can this ever be empty? We validate against this and we provide the implicit value, based on plug or slot name, when reading snap.yaml.